sdks/python/apache_beam/internal/apiclient.py

```diff
@@ -165,7 +163,7 @@ def __init__(self, packages, options, environment_version):
           dataflow.Package(
               location='%s/%s' % (
                   self.google_cloud_options.staging_location.replace(
-                      'gs:/', STORAGE_API_SERVICE),
```

Comment:
From PR description - "staging_location and temp_location - In Java, tempLocation is the default value for stagingLocation. In Python I've changed the behavior to be the same.".

Was this not implemented ? I don't see the change here (apiclient.py:328)

Seems like some other changes mentioned in the PR description are not implemented here as well. Can you pls update the PR description to accurately describe this change ?